File: app/src/main/java/pl/elpassion/door/githubclient/service/github/UserRepositoriesService.java
package pl.elpassion.door.githubclient.service.github;

import java.util.List;

import pl.elpassion.door.githubclient.service.github.response.Repository;
import retrofit2.http.GET;
import retrofit2.http.Path;
import rx.Observable;


public interface UserRepositoriesService {

    @GET("users/{user}/repos")
    Observable<List<Repository>> getUserRepositories(@Path("user") String userName);

}

File: app/src/main/java/pl/elpassion/door/githubclient/service/github/UserRepositoriesService.kt
package pl.elpassion.door.githubclient.service.github

import pl.elpassion.door.githubclient.service.github.response.Repository
import retrofit2.http.GET
import retrofit2.http.Path
import rx.Observable


interface UserRepositoriesService {

    @GET("users/{user}/repos")
    fun getUserRepositories(@Path("user") userName: String): Observable<List<Repository>>
}